lib/npm/config.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const execa = require('execa');
-const camelcaseKeys = require('camelcase-keys');
+const camelCaseKeys = require('camelcase-keys');
 
 module.exports = async function (cwd) {
   const child = execa.sync('npm', ['config', 'list', '--json'], {
```

Comment:
Why `execa.sync` in an async function?

I realize that execa is managing subprocesses, but using `child` here is a bit confusing to read. Would `result` be better?

```js
console.log(result.stdout);
// etc
```

Reply:
I can do a follow up PR that changes this, but given it isn't part of this existing PR I would like to keep it seperate.

re: `result` I feel that is ambiguous, but `child` isn't really accurate either as it represents the outcome of the child processes completion and no longer the child itself. Reading further, it seems that execa refers to this value as a [childProcessResult](https://github.com/sindresorhus/execa#childProcessResult) which is a mouth full.

Given that `result` does seem appropriate.

Reply:
👍